Makefile

```diff
@@ -277,7 +277,7 @@ help:  # Display this help
 
 .PHONY: generate
 generate: ## Run all generate targets
-	$(MAKE) generate-modules generate-manifests generate-go-deepcopy generate-go-conversions generate-flavors
+	$(MAKE) generate-modules generate-manifests generate-vm-operator-manifest generate-go-deepcopy generate-go-conversions generate-flavors
```

Comment:
TBD if we always want to run this target in pre-submit (takes a bit of time, but I don't thing very long)

@fabriziopandini @chrischdi WDYT?

Reply:
I would say no, creates additional requests to clone a repo.

Reply:
I'm fine either way. It's a trade-off between always running these comments vs potentially the files get outdated.

Given that these files should only change when we bump vm-operator and then we'll change the file anyway it should be fine to drop this target from generate / verify-gen.

I can change the output-file to contain the version. This way it becomes really hard to miss when reviewing vm-operator bumps of the vsphere.yaml

Reply:
Done